Assets/Scripts/Gameplay/GameplayObjects/Character/ClientCharacter.cs

```diff
@@ -166,8 +162,8 @@ public override void OnNetworkSpawn()
 
                     if (m_ServerCharacter.TryGetComponent(out ClientInputSender inputSender))
                     {
-                        // TODO: revisit; anticipated actions would play twice on the host
-                        if (!IsServer)
+                        // anticipated actions will only be played on non-host, owning clients
+                        if (!m_IsHost)
```

Comment:
If BR ever gets ported to DGS, wouldn't this have issues?
Why doesn't !IsServer work?